### DISCUSSION TITLE
Subdivision, and loop cuts

### DISCUSSION THREAD

**Discussion by Davkey** [blackears/cyclopsLevelBuilder#189] [Ideas]
Hello, I am using this to make a game that will use stuff like vertex lighting and colors, I am more used to unity's probuilder, I think there should be a subdivide/loop cut feature for cyclops if it is a map editor, thanks!

**Comment by blackears** [comment]
At the moment the editor only works with convex blocks, so there is no way to add loop cuts.  I do want to move to a general mesh editor in the future, but that will require making some changes to the core engine first.
